packages/umi-server/src/index.ts

```diff
@@ -4,19 +4,23 @@ import { load } from 'cheerio';
 import compose from './compose';
 import _log from './debug';
 import { nodePolyfillDecorator, injectChunkMaps, _getDocumentHandler } from './utils';
+import React from 'react'
```

Comment:
```suggestion
import { ReactChild } from 'react';
```

然后安装一个 `@types/react`